README.md

```diff
@@ -51,7 +51,9 @@ In addition to specifying your publisher credentials directly within each build
 
 6. Select this endpoint using the name you chose in the previous step whenever you add either the **App Store Release** or **App Store Promote** tasks to a build or release definition.
 
-### Supporting Two-Factor Verification
+### Two-Factor Authentication
```

Comment:
Please update https://github.com/Microsoft/app-store-vsts-extension/blob/master/docs/vsts-README.md to be in sync as well